de4a-kafka-client/src/main/java/eu/de4a/kafkaclient/DE4AKafkaManager.java

```diff
@@ -118,6 +128,28 @@ private DE4AKafkaManager ()
     return ret;
   }
 
+  /**
+   * Init the global Kafka HTTP Client if HTTP is used instead of Kafka TCP Streams
+   *
+   * @return The non-<code>null</code> HttpURLConnection to be used.
+   *
+   */
+
+  @Nonnull
+  public static HttpURLConnection createUrlConnection() throws IOException {
+
+      HttpURLConnection conn;
```

Comment:
I'd prefer to use Apache http Client, simply because it can more easily and finegraind be configured in regards to SSL and Proxy. And this dependency will be part of the Connector which has the Apache HttpClient anyway...

Reply:
ok. will fix this one

Reply:
if you like, there is an abstraction in com.helger.web:ph-httpclient which is like this:
    try (final HttpClientManager aHttpClientMgr = HttpClientManager.create (m_aHttpClientSettings))
    {
      return aHttpClientMgr.execute (aRequest, aResponseHandler);
    }

and you configure all the things you need in the HttpClientSettings